### ISSUE TITLE
Add `sst nuke` command

### ISSUE THREAD

**Issue by garysassano** [sst/ion#672]
It would be nice to have something similar to [aws-nuke](https://github.com/rebuy-de/aws-nuke) for purging all resources created by SST in an AWS account (the parameters in SSM, the states in S3, etc.)

**Comment by jschuur** [comment]
There's the '[removal](https://ion.sst.dev/docs/reference/config/#removal)' option in `sst.config.ts`, which defaults to 'retain' for production:

```ts
 removal: input?.stage === 'production' ? 'retain' : 'remove',
```

It would be handy to be able to override that in the regular `sst remove` command.

Does 'remove' still leave some resources?

**Comment by garysassano** [comment]
> Does 'remove' still leave some resources?

Yes, `sst remove` retains some resources in your AWS account which are used internally by SST.

**Comment by jschuur** [comment]
> > Does 'remove' still leave some resources?
> 
> Yes, `sst remove` retains some resources in your AWS account which are used internally by SST.

I was referring to explicitly setting `removal: 'remove'` in `sst.config.ts` not the `sst remove` CLI command. The remove command uses this setting to decide how much to remove.

The docs say 'remove: Remove all your resources on remove.', but I haven't validated if  'all' doesn't include internal ones.
